private/model/api/api.go

```diff
@@ -395,6 +395,11 @@ func (a *API) APIOperationGoCode(op *Operation) string {
 		a.AddSDKImport("private/protocol")
 	}
 
+	if op.HasEndpointARN {
+		a.AddImport("fmt")
+		a.AddSDKImport("service", a.PackageName(), "internal", "arn")
```

Comment:
this should use the `a.ImportPath()` instead of `service, a.PackageName` with `AddImport`. The `service` component can be passed into the `gen-api` utility as a cli command.